tools/metamorpheus/metamorpheus_calibration.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="metamorpheus_calibration" name="MetaMorpeus Calibration" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" python_template_version="3.5">
+    <description>identify peptides by their calibrate the m/z values of all peaks in the spectra</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+@PREPARE_INPUTS@
+@MERGE_TOMLS@
+metamorpheus
+-t tasksettings.toml
+## -s #echo ' '.join($spectrum_names)#
+-s $inputs_dir
+-d #echo ' '.join($database_names)#
+-o output
+    ]]></command>
+    <configfiles>
+        <configfile name="default_toml"><![CDATA[#slurp
+@DEFAULT_CALIBRATION_TOML@
+]]></configfile>
+        <configfile name="user_toml">
+@CALIBRATE_CONFIG@
+@COMMON_CONFIG@
+        </configfile>
+        <configfile name="task_settings_toml">#slurp
+        </configfile>
+    </configfiles>
+    <inputs>
+        <expand macro="input_spectrums" multiple="true"/>
+        <expand macro="input_databases"/>
+        <expand macro="input_toml"/>
+        <expand macro="calibrate_params"/>
+        <expand macro="common_params"/>
+    </inputs>
+    <outputs>
+        <expand macro="output_settings_toml"/>
+        <data name="results" format="txt" label="${tool.name} on ${on_string} results.txt" from_work_dir="output/allresults.txt"/>
+        <data name="prose" format="txt" label="${tool.name} on ${on_string} prose.txt" from_work_dir="output/Task1CalibrationTask/prose.txt"/>
+        <expand macro="calibrated_outputs"/>
+    </outputs>
+    <tests>
+        <test>
```

Comment:
can you add `expect_num_outputs` .. which is always a good idea if there are outputs with filters.